dns_server/src/crawler_p2p/crawler/tests/mod.rs

```diff
@@ -129,13 +129,13 @@ fn randomized(#[case] seed: Seed) {
         })
         .collect::<Vec<_>>();
 
-    let added_count = rng.gen_range(0..5);
-    let added_nodes = nodes.choose_multiple(&mut rng, added_count).cloned().collect();
+    let reserved_count = rng.gen_range(0..5);
```

Comment:
It's always better to randomize this 5 and that 10 and create a model that covers them both with the relation between them. This applies to all magic numbers.

Reply:
Sorry, I'm not sure what you mean. I just wanted to allow testing of corner cases and this should be covered here.

Reply:
Instead of 5, use a random number. Instead of 10, use a random number. Define the boundaries of the random number so it covers a wide range over time.

Reply:
Could you please write how it should look? I think it would be easier for me to understand.

Reply:
As agreed in the call. This is OK.